src/folding/hypernova/circuit.rs

```diff
@@ -21,7 +22,7 @@ impl<F: PrimeField> SumMulsGammaPowsEqSigmaGadget<F> {
     /// # Arguments
     /// - `sigmas`: vector of $\sigma_j$ values
     /// - `eq_eval`: the value of $\tilde{eq}(x_j, x^{\prime})$
-    /// - `gamma`: a `GammaVar`, which supports a `pow` method, representing $\gamma$
+    /// - `gamma`: value $\gamma$
```

Comment:
I'm surprised to see this change, I thought that this was already done in the previous PR and already included in the pointed branch `feature/hypernova-nimfs-verifier-circuit` (https://github.com/privacy-scaling-explorations/folding-schemes/blob/11066b15f9ef4c97dba2b05608b0d093405ae0f7/src/folding/hypernova/circuit.rs#L24).
Is the branch of the current PR not departing from latest version of `feature/hypernova-nimfs-verifier-circuit`?

Reply:
to be clear, this is not a request of change, just that I'm curious with the current diff hehe

Reply:
I rebased feature/hypernova-nimfs-verifier-circuit` twice on this branch before and after the merge, this might explain why the change for this updated comment is here! Sorry, this is kinda confusing.

Reply:
oh cool, make sense. Was asking to learn, thanks ^^